src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/PushController.cs

```diff
@@ -28,19 +29,38 @@ public class PushController : ControllerBase
         private readonly AuthorizationHelper _authorizationHelper;
         private readonly PlatformSettings _platformSettings;
 
+        private readonly IMemoryCache _memoryCache;
+        private readonly MemoryCacheEntryOptions orgSubscriptioncacheEntryOptions;
```

Comment:
differing from the naming standard with _ as prefix

Reply:
fixed